pom.xml

```diff
@@ -603,12 +603,6 @@
                 <version>2.1.5.3</version>
             </dependency>
 
-            <dependency>
```

Comment:
Commit title is slightly too long. Please change first line to something like 
```
Complete removal of joda-to-java-time-bridge
```

Reply:
Sure

Reply:
Done